packages/insured-bridge-relayer/src/index.ts

```diff
@@ -52,14 +52,25 @@ export async function run(logger: winston.Logger, l1Web3: Web3): Promise<void> {
     // Construct a web3 instance running on L2.
     const l2Web3 = getWeb3ByChainId(config.activatedChainIds[0]);
     const latestL2BlockNumber = await l2Web3.eth.getBlockNumber();
-    const l2StartBlock = latestL2BlockNumber - config.l2BlockLookback;
+    const l2StartBlock = Math.max(0, latestL2BlockNumber - config.l2BlockLookback);
+    const l2EndBlock = config.l2EndBlock > 0 ? config.l2EndBlock : null;
 
     const l2Client = new InsuredBridgeL2Client(
       logger,
       l2Web3,
       await l1Client.getL2DepositBoxAddress(config.activatedChainIds[0]),
       config.activatedChainIds[0],
-      l2StartBlock
+      l2StartBlock,
+      l2EndBlock
+    );
+
+    // Update the L2 client and filter out tokens that are not whitelisted on the L2 from the whitelisted
+    // L1 relay list.
+    await l2Client.update();
+    const filteredWhitelistedRelayL1Tokens = await pruneWhitelistedL1Tokens(
```

Comment:
if you use a smaller variable name this could fit on one line. how about `filteredL1Whitelist`

Reply:
done